common/djangoapps/track/views/__init__.py

```diff
@@ -78,6 +81,36 @@ def user_track(request):
 
     return HttpResponse('success')
 
+perflog = logging.getLogger("perflog")
+
+
+def performance_log(request):
+    """
+    Log when POST call to "performance" URL is made by a user.
+    Request should provide "event" and "page" arguments.
+    """
+    page = _get_request_value(request, 'page')
+
+    with eventtracker.get_tracker().context('edx.course.browser', contexts.course_context_from_url(page)):
+        event = {
+            "ip": _get_request_header(request, 'REMOTE_ADDR'),
+            "referer": _get_request_header(request, 'HTTP_REFERER'),
+            "accept_language": _get_request_header(request, 'HTTP_ACCEPT_LANGUAGE'),
+            "event_source": "browser",
```

Comment:
I'll check for clarity -- I think the answer is no, and in any case, the data will still be valid. We're also collecting UA which I believe is different for the app.